### PR TITLE
Add configuration to support "social sharing" feature

### LINKED ISSUE #22: Page Sharing
**What:**
Add support for buttons on the hero section to being able to share the website.

**Acceptance Criteria**
- [x] Share on Twitter
- [x] Share on Facebook
- [x] Share on LinkedIn **Optional**

re #21

### PR DESCRIPTION
**Pull request by duranmla**
**What:**
Allow users to share the page

**Why:**
Closes #22 

**How:**
Add integration with "[addthis.com](https://www.addthis.com/)" service for quick social actions management. The customisation of the widget to render is being done within their admin panel. Any further customisations or need please take also a look to https://www.addthis.com/academy/the-addthis_config-variable/

**Extras:**
- There is a minor fix at 173d1ee that allow us to keep the hero section as a truly one page scroll. _(read commit for more information)_

## Screenshots
image

https://www.loom.com/share/59c4ad82d3a044d898e2bdab44dadcca

**IMPORTANT:** LinkedIn integration seems to not work properly, in any case, we can decide what to show and what not from the addthis admin panel

### CODE REVIEW

**Comment by duranmla** [comment]
This one will need the fixes at https://github.com/debtcollective/student-debt-campaign/pull/60 in order for the build to successfully pass

File: gatsby-browser.js [hunk 1]
```diff
@@ -1 +1,7 @@
 export { wrapRootElement } from './src/apollo/wrap-root-element'
+
+export const onInitialClientRender = () => {
+  const addThisScript = document.createElement('script')
+  addThisScript.src = `//s7.addthis.com/js/300/addthis_widget.js#pubid=${process.env.GATSBY_ADDTHIS_PUB_ID}`
```
Comment (orlando) [inline gatsby-browser.js]: @duranmla let's add a check here to only render this if `GATSBY_ADDTHIS_PUB_ID` variable is present

File: src/components/DataDuesAction/index.js [hunk 2]
```diff
@@ -13,10 +13,9 @@ import {
   unknown,
   validationSchema
 } from './schema'
-import './styles.scss'
 
 const DataDuesHeader = () => (
-  <>
+  <div id="data-dues-page" className="data-dues-page">
```
Comment (orlando) [inline src/components/DataDuesAction/index.js]: @duranmla this is the data dues component, the page should be in the page folder. `DataDuesHeader` only contains the text before the form. Let's remove this

File: src/styles/layout.scss [hunk 3]
```diff
@@ -20,7 +20,3 @@
     padding-top: ($spacer * 6);
   }
 }
-
-.main {
```
Comment (orlando) [inline src/styles/layout.scss]: @duranmla what's the reason behind this change? I changed to move this responsibility to the Layout (which is used in all pages)

File: gatsby-browser.js (hunk 1 above)
Comment (duranmla) [reply gatsby-browser.js]: It indeed only renders when the env variable is rendered. But we can avoid to create the script tag when the variable is not there.

File: src/styles/layout.scss (hunk 3 above)
Comment (duranmla) [reply src/styles/layout.scss]: Check the PR description. I made special emphasis in order for you to know why. 173d1ee also the commit message have information about it.

Comment (duranmla) [reply src/styles/layout.scss]: Indeed, it would be great if we can have the responsibility of that space in a "layout" level but I was aiming to fix this as easy as I can and if this start to be a "problem to maintain" or you have a better idea that also meets the expectations without break the hero we can do it.

File: gatsby-browser.js (hunk 1 above)
Comment (duranmla) [reply gatsby-browser.js]: Addressed here 430f4ec
